homeassistant/components/yamaha/media_player.py

```diff
@@ -99,15 +101,18 @@
 class YamahaConfigInfo:
     """Configuration Info for Yamaha Receivers."""
 
-    def __init__(self, config: ConfigType, discovery_info: DiscoveryInfoType) -> None:
+    def __init__(
+        self, config: ConfigType, discovery_info: DiscoveryInfoType | None
+    ) -> None:
         """Initialize the Configuration Info for Yamaha Receiver."""
+        _LOGGER.error("%s", config)
```

Comment:
Stale debug statement?

```suggestion
```

Reply:
That was for debugging - forgot to remove it. Thanks